tasks/main.yml

```diff
@@ -18,6 +18,7 @@
 
 - include: authorized_keys.yml
   tags: ['ssh']
+- include: sshlog.yml
```

Comment:
I think that would be better to check a variable to include or not this tasks, and use `when` in the include `statement`,  to do this conditional:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#applying-when-to-roles-imports-and-includes